node/harmony/node.go

```diff
@@ -1107,8 +1107,8 @@ func New(
 	node.serviceManager = service.NewManager()
 
 	// delete old pending crosslinks
-	if node.Blockchain().ShardID() == shard.BeaconChainShardID {
-		ten := big.NewInt(10)
+	ten := big.NewInt(10)
+	if node.Blockchain().ShardID() == shard.BeaconChainShardID && node.Blockchain().CurrentBlock().Epoch().Cmp(ten) > 0 {
```

Comment:
in localnet the crosslink hardfork epoch happens before 10th epoch, you can use `func (c *ChainConfig) IsCrossLink(epoch *big.Int) bool` to improve your code

Reply:
<img width="783" alt="image" src="https://github.com/user-attachments/assets/0d464579-bb52-4947-bb53-a70a120a634f">
no, it's not. What means 10 then?

Reply:
it is [2 for localnet](https://github.com/harmony-one/harmony/blob/main/internal/params/config.go#L281), and [186 for mainnet](https://github.com/harmony-one/harmony/blob/main/internal/params/config.go#L44)

Reply:
done

Reply:
10 is magic number, you should use `IsCrossLink ` method instead.